wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -0,0 +1,104 @@
+
+- [osu!(lazer)](https://github.com/ppy/osu)
+- [The osu! website](https://github.com/ppy/osu-web/)
+- [Stable osu! build issues](https://github.com/ppy/osu-stable-issues)
+
+#### Issue labels
+
+On GitHub, pull requests and issues may be tagged and classified by using [labels](https://github.com/ppy/osu-wiki/labels), which show different aspects of a pull request or issue. Labels are set by the wiki maintainers and are informational and typically self-explanatory. While they require no action from a user's perspective, red issue labels serve as a reminder or call to action for other maintainers:
+
+- `rule change`: the change affects an existing set of rules, such as [Ranking Criteria](/wiki/Ranking_Criteria), and needs to be reviewed by the area's owner
+- `blocked`: the change has issues which must be resolved before proceeding, or depends on another issue which must be resolved first
+- `needs native review`: the translation needs to be checked by a person fluent in the respective language. Alternatively, during the merge process it means that no such review took place
+- `needs rebase`: the pull request has too many small unstructured commits, which need to be rebased and worded in a better way
+
+### Links and redirects
+
+Most osu! wiki articles have alternative links, which are set up using the [`redirect.yaml`](https://github.com/ppy/osu-wiki/blob/master/wiki/redirect.yaml) file. The redirects are meant to be used outside the osu! wiki, for example, on forums, or in the [chat](/wiki/Chat_Console), where they could be quickly turned into an inline reference:
+
+```
+According to [[RC]], this is forbidden.
+```
+
+When adding redirects for a new or existing article, make sure to check the following:
+
+- The redirect should be designed for use and be conscise—don't add it for the sake of adding.
+- The redirect should not conflict with existing redirects.
+
+<!-- TODO: should probably briefly mention external links https://github.com/ppy/osu-web/issues/8086 and footnotes https://github.com/ppy/osu-wiki/issues/4911#issuecomment-893959588 once they are implemented -->
+
+### CI checks
+
+The osu! wiki repository uses [continuous integration](https://docs.github.com/en/actions/guides/about-continuous-integration) (CI) to automatically check incoming pull requests for different common errors. The list of checks is configured in the [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file and includes several custom plugins written by the osu! wiki maintainers.
+
+The CI checks are run automatically on every commit of a recurring contributor. In order to have their pull requests merged, contributors are expected to fix errors reported by the CI. To see the [status of checks](img/ci-status.png), do the following:
```

Comment:
Is not showing the image in the article intentional?

Reply:
yes, it's bulky. want to place an infobox here when they're ready

Reply:
```suggestion
The CI checks are run automatically per every commit push to the feature branch. In order to have their pull requests merged, contributors are expected to fix errors reported by the CI. To see the [status of checks](img/ci-status.png), do the following:
```